src/js/controllers/mapController.ts

```diff
@@ -127,7 +129,12 @@ export class MapController {
       .when(
         () => {
           store.dispatch(isMapReady(true));
-          this._mapview?.on('click', event => {
+          this._mapview.popup.highlightEnabled = false;
+          this._mapview.on('click', event => {
+            //TODO: We need a better loading handling, probably a spinner!
+            //clean active indexes for data tab and activeFeatures
+            store.dispatch(setActiveFeatures([]));
```

Comment:
@vaidotasp My only Q here - do we want to `zero-out` the activeFeatures here if we are resetting them in the `resolveClientPromisesWithErrors` that is also triggered here in the "_mapview.on('click'"  ?